src/screens/profile_details/hooks.ts

```diff
@@ -59,6 +59,21 @@ export const useProfileDetails = () => {
     }
   }, [state.desmosProfile]);
 
+  useEffect(() => {
+    if (state.desmosProfile) {
+      const dtagConnections = state.desmosProfile.connections;
+      const dtagConnectionsNetwork = dtagConnections.map((x) => { return x.network; });
+      dtagConnectionsNetwork.push('desmos');
+      const chainPrefix = chainConfig.prefix.account;
+      const containNetwork = dtagConnectionsNetwork.includes(chainPrefix);
+      if (!containNetwork) {
+        handleSetState({
+          exists: false,
+        });
+      }
+    }
+  }, [state.desmosProfile]);
+
```

Comment:
If we are checking the chainPrefix wouldn't it be easier to use `x.identifier` instead of `x.network`? This way you can even remove line 66

Reply:
I see.